src/DataDepsGenerators.jl

```diff
@@ -54,6 +52,16 @@ function find_metadata(repo, dataname, shortname)
     )
 end
 
+function data_shortname(repo, shortname::Void, fullname)
```

Comment:
Colons `:` are still showing up in the names. Which is not legal.
Also unescaped `"`.
`data_shortname` should strip both
Not sure why it isn't.